packages/valory/skills/decision_maker_abci/behaviours/base.py

```diff
@@ -618,6 +624,9 @@ def _write_benchmark_results(
             )
             return
 
+        if liquidity_info.l0_start is None:
+            self.context.logger.info("No market liquidity information.")
```

Comment:
I see you are using liquidity_info on line 663-666. If this is None, shouldnt we handle this case differently?

Reply:
Corrected